sklearn/feature_extraction/text.py

```diff
@@ -479,6 +479,44 @@ class HashingVectorizer(BaseEstimator, VectorizerMixin, TransformerMixin):
     dtype : type, optional
         Type of the matrix returned by fit_transform() or transform().
 
+    Examples
+    --------
+    >>> from sklearn.feature_extraction.text import CountVectorizer
+    >>> corpus = [
+    ...     'This is the first document.',
+    ...     'This is the second second document.',
```

Comment:
Maybe we should provide users with a meaningful sentence, so remove one ``second``?

Reply:
Good point. I'll change the sentence to be more meaningful.